src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,15 +1,23 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private static final String START_WORK_MESSAGE = "Bulldozer started to work";
+
+    private String trackType;
```

Comment:
Don't forget about access modifiers for your custom fields in `Bulldozer` class. The field `trackType` should be accessed through special methods (getters and setters), so it's better to make it private to adhere to the principles of encapsulation.